client/my-sites/purchases/payment-methods/index.tsx

```diff
@@ -72,6 +72,7 @@ export function PaymentMethods( { siteSlug }: { siteSlug: string } ): JSX.Elemen
 				brandFont
 				className="payment-methods__page-heading"
 				headerText={ titles.sectionTitle }
+				subHeaderText={ translate( 'Add or delete payment methods for your account.' ) }
```

Comment:
This one is good - even though it's the site-specific view, payment methods are managed for the entire account.

Reply:
`client/my-sites/purchases/main.tsx` is corrected now as well.

<img width="530" alt="Screen Shot 2021-07-23 at 12 32 51 PM" src="https://user-images.githubusercontent.com/8002138/126832602-e843d196-4693-476e-b8ff-677c2e7a0590.png">

Marking as resolved.